package.json

```diff
@@ -55,12 +55,12 @@
     "@types/pako": "^1.0.7",
     "@types/react": "^17.0.80",
     "@types/react-dom": "^17.0.25",
-    "@types/webpack": "^5.28.5",
+    "@types/webpack": "^4.41.40",
```

Comment:
>"@types/webpack": "^4.41.40", [](http://example.com/codeflow?start=4&length=29)

Why is this one going backwards?

Reply:
I looked into this and 4.41.40 is the latest version release (13 days ago) vs 5.28.5 hasn't been updated in over 1 year. That's why this one in particular is going backwards